spec/models/vote_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+
+describe Vote do
+  describe "validations" do
+    describe "value validation" do
+      it "only allows -1 or 1 as values" do
+        expect(@vote.value).to eq(true || false)
```

Comment:
In this case, we want to test that a `Vote`'s `value` can only be initialized to **1** or **-1**. This means that after initializing a `Vote` object with a different number, if you try to `save` it, would return false.

`eq` should only be expecting one thing, depending of what you are testing. For example, if you initialized a `Vote` object with the value **1**, then your expectation should be that saving the vote `eq(true)`. On the other hand, if you initialize a `Vote` object with the value **2**, then your expectation should be `eq(false)`.